teuthology/task/ansible.py

```diff
@@ -213,9 +224,13 @@ def _build_args(self):
         """
         Assemble the list of args to be executed
         """
-        fqdns = [r.name.split('@')[-1] for r in self.cluster.remotes.keys()]
+        fqdns = [r.hostname for r in self.cluster.remotes.keys()]
+        # Assume all remotes use the same username
+        user = self.cluster.remotes.keys()[0].user
```

Comment:
Just a note: we should add the teuthology_user to teuthology.config sometime. We'll eventually need that.